### DISCUSSION TITLE
Fusion power volume integration

### DISCUSSION THREAD

**Discussion by jmorris-uk** [ukaea/PROCESS#3254] [Ideas]
# Original issue (https://github.com/ukaea/PROCESS/issues/766)

The volume element is included in the 2*rho factor: 

This seems a bit crude as it applies only for circular flux surfaces. It is true that in the core where the fusion takes place the local elongation and triangularity are small. Would it be practical to use a more accurate formula taking account of elongation of the flux surface?

# Papercut review

Need to decide upon a method for more accurately integrate the fusion power for non-circular plasmas before proceeding.

Any method developed here will be superseded once we couple/implement an equilibrium solver into PROCESS again.

The original issue has been closed and discussion on this model moved here.

**Comment by chris-ashe** [comment]
I think since this boils down to the density and temperature profiles used, both of which are only dependant on the radial position in the plasma and are identical for the inboard and outboard. This means doing any volume integration for a shape with vertical and radial symmetry with work with the profiles, (circle, ellipse etc) https://ukaea.github.io/PROCESS/physics-models/profiles/plasma_density_profile/#derivation
We could either create 2 separate profiles for the inboard and outboard of the plasma or find a fitting function for how the profile scales with normalized height.
